src/plugin/vault/etc/test-vault.sh

```diff
@@ -35,13 +35,6 @@ vault write $PLUGIN_PATH/config \
 	pachd_address="${ADDRESS:-127.0.0.1}:30650"
 go build -o /tmp/vault-plugins/$PLUGIN_NAME src/plugin/vault/main.go
 
-
-# Test login before admin token is set
-# vault write $PLUGIN_PATH/login username=tweetybird || true
-
 # Test login (failure/success):
-vault write $PLUGIN_PATH/login username=bogusgithubusername || true
-vault write $PLUGIN_PATH/login username=daffyduck ttl=125s
-
-# To renew, use the 'token' field
-# vault token renew ee9ab3ef-e398-3437-d382-b7aaf02f32e1
+vault write -f $PLUGIN_PATH/login/github:bogusgithubusername
+vault write -f $PLUGIN_PATH/login/github:daffyduck ttl=125s
```

Comment:
I know I was the one who put this in there ... and while I think its funny that a real GH user exists called daffyduck ... we should reign in any assumptions in our tests ... at least we could make the username one of us.

Reply:
Fixed. I used jdoliner and JoeyZwicker